02-peer-protocol.md

```diff
@@ -894,40 +916,43 @@ any message), they are independent of requirements here.
 ### Requirements
 
 A node MUST handle continuing a previous channel on a new encrypted
-transport.  On disconnection, a node MAY forget nodes which have not
-sent or received an `accept_channel` message, and MAY forget nodes
-which have not sent `funding_locked` after a reasonable timeout.
+transport.
+
+On disconnection, the funder MUST remember the channel for
+reconnection if it has broadcast the funding transaction, otherwise it
+MUST NOT.
```

Comment:
This should probably allow for unilateral closes after a timeout as well, i.e., the situation where funder broadcasts, disconnect, timeout -> unilateral close, reconnect. In that case the reconnect must be aborted.